app/main.py

```diff
@@ -1,44 +1,64 @@
 import time
-
+import asyncio
+from typing import Any, Awaitable
```

Comment:
Imports should be grouped and separated by a blank line according to standard library, external libraries, and custom imports. This practice enhances both code readability and maintainability. 
Read more about topic [here](https://peps.python.org/pep-0008/#imports)